docs/ros/ros/installation/remote_pc.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Remote PC setup
+sidebar_label: Remote PC
+sidebar_position: 2
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+### Operating System (OS) {#operating-system}
+
+ROS 2 Humble uses [Ubuntu 22.04](https://releases.ubuntu.com/jammy/) as its Tier 1 operating system. Though other operating systems are supported,
+it is highly recommended to use Ubuntu 22.04. For the remote PC, Ubuntu 22.04 Desktop should be installed.
+
+### Networking
+
+The remote PC should be connected to the same Wi-Fi network as the robot PC, or should be connected via ethernet. If connected over ethernet,
+set your static IP to the `192.168.131` subnet, with an IP address that won't interfere with other devices on the network, such as `192.168.131.99`.
```

Comment:
"your static IP"? Is this the IP of the the Remote Computer? If so, can we update to clarify?